api/all/src/main/java/io/opentelemetry/api/trace/TraceFlags.java

```diff
@@ -15,15 +15,6 @@
  */
 @Immutable
 public interface TraceFlags {
-  /**
-   * Returns the length of the lowercase hex (base16) representation of the {@link TraceFlags}.
-   *
-   * @return the length of the lowercase hex (base16) representation of the {@link TraceFlags}.
-   */
-  static int getLength() {
```

Comment:
I don't think we should remove this at this point. It's not harmful to have it here, so let's keep it as-is.